universal-application-tool-0.0.1/app/repository/ProgramRepository.java

```diff
@@ -47,6 +52,50 @@ public Program updateProgramSync(Program program) {
     return program;
   }
 
+  public void updateQuestionVersions(Program program) {
+    Transaction transaction = ebeanServer.beginTransaction();
+    try {
+      updateQuestionVersions(program, transaction);
+      ebeanServer.commitTransaction();
+    } finally {
+      ebeanServer.endTransaction();
+    }
+  }
+
+  public void updateQuestionVersions(Program program, Transaction transaction) {
```

Comment:
Suggestion (blocking): please add javadoc for this method describing what it does